.prettierrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "overrides": [
+    {
+      "files":[
+        "**/*.js"
+      ],
+      "options": {
+        "singleQuote": true,
+        "semi": false,
+        "trailingComma": "es5"
```

Comment:
Options were to keep the formatting to prevent diffs with this, but could be updated later to reduce future diffs